ebos/eclcpgridvanguard.hh

```diff
@@ -191,6 +192,8 @@ public:
             //distribute the grid and switch to the distributed view.
             {
                 const auto wells = this->schedule().getWellsatEnd();
+                auto& eclState = static_cast<ParallelEclipseState&>(this->eclState());
```

Comment:
These two lines seem to be left overs from a rebase. Please delete as this in now two lines below in the try-clause

Reply:
Now removed.